src/tally.F90

```diff
@@ -2224,6 +2239,295 @@ subroutine score_surface_current(p)
   end subroutine score_surface_current
 
 !===============================================================================
+! APPLY_DERIVATIVE_TO_SCORE multiply the given score by its relative derivative
+!===============================================================================
+
+  subroutine apply_derivative_to_score(p, t, i_nuclide, atom_density, &
+                                       score_bin, score)
+    type(Particle),    intent(in)    :: p
+    type(TallyObject), intent(in)    :: t
+    integer,           intent(in)    :: i_nuclide
+    real(8),           intent(in)    :: atom_density   ! atom/b-cm
+    integer,           intent(in)    :: score_bin
+    real(8),           intent(inout) :: score
+
+    integer :: l                    ! loop index for nuclides in material
+    logical :: scoring_diff_nuclide
+
+    select case (t % deriv % variable)
+
+    case (DIFF_DENSITY)
+      select case (t % estimator)
+
+      case (ESTIMATOR_ANALOG)
+        if (materials(p % material) % id == t % deriv % diff_material) then
```

Comment:
So I guess there are no problems with void materials since the functionality doesn't work with tracklength estimators. Is that correct?